src/main/py/drawing/coordinates.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
+"""
+Module to manage coordinates class
+
+Coordinate class allows us to create a coordinates with x,y values
+"""
+
+
+class Coordinates:
+    """
+    Class to manage coordinates
+    """
+
+    def __init__(self, x: int, y: int):
+        """
+        Initializes an instance of the Coordinates class.
+        - x (int): x-coordinate
+        - y (int): y-coordinate
+        """
+        self.x = self.normalize_coordinate(x)
+        self.y = self.normalize_coordinate(y)
+
+    def normalize_coordinate(self, coordinate):
+        """
+        If coordinate < 0, make it 0.
+        """
+        if coordinate < 0:
+            return 0
+        return coordinate
+
+    def create_bounds(self, width, height):
```

Comment:
I didn't see tests for this method. One should be added that tests the various boundary conditions, such as:
* x/y is the last pixel
* x/y is the next to last pixel
* x/y is at 0,0